conf/genomes/crick/genomes.config

```diff
@@ -0,0 +1,51 @@
+/*
+ * -------------------------------------------------------
+ *  Nextflow config file for Genomes available on CAMP
+ * -------------------------------------------------------
+ * For example:
+ * nextflow run nf-core/rnaseq \
+ *     --input samplesheet.csv \
+ *     --genome GRCh38.95.CAMP \
+ *     -profile crick \
+ */
+
+params {
+  genomes {
+    'GRCh38.95.CAMP' {
+      fasta       = "/camp/svc/reference/Genomics/babs/homo_sapiens/ensembl/GRCh38/release-95/genome/Homo_sapiens.GRCh38.dna_sm.primary_assembly.fa"
```

Comment:
You could add a `params.genomes_base` for `/camp/svc/reference/Genomics/babs` to simplify the setup and make it easier to steal for other people ;-)

Reply:
I did think about doing that but I figured that I would try and leave it as vanilla as possible for now. Also, having explicit paths is much easier to read and update. Would still need to create your own custom configs with these paths if you wanted to steal anyway! :)